app/src/main/java/com/example/mvppattern/ui/main/MainContract.kt

```diff
@@ -3,33 +3,34 @@ package com.example.mvppattern.ui.main
 import android.content.ContentValues
 import android.content.Context
 import com.example.mvppattern.data.Student
+import com.example.mvppattern.data.repository.DBManager
 
 interface MainContract {
     interface View {
-        fun onShowAllStudentSuccess(mList: ArrayList<Student>)
-        fun onShowAllStudentFail()
-        fun onDeleteStudentSuccess(position: Int)
-        fun onDeleteStudentFail()
-        fun onUpdateStudentSuccess()
-        fun onUpdateStudentFail()
+        fun showAllStudentSuccess(mList: ArrayList<Student>)
+        fun showAllStudentFail()
+        fun deleteStudentSuccess(position: Int)
+        fun deleteStudentFail()
+        fun updateStudentSuccess()
+        fun updateStudentFail()
```

Comment:
suýt tốt, không hề có success hay fail gì nhé, chỉ có showMessage hay updateStudents hay gì đó thôi
```suggestion
        fun showAllStudentSuccess(students: List<Student>)
```